--- a/webactivity.py
+++ b/webactivity.py
@@ -33,9 +33,12 @@
 import locale
 import cairo
 
-# HACK: Needed by http://dev.sugarlabs.org/ticket/456
-import gnome
-gnome.init('Hulahop', '1.0')
+try:
+    # HACK: Needed by http://dev.sugarlabs.org/ticket/456
+    import gnome
+    gnome.init('Hulahop', '1.0')
+except:
+    pass
 
 from sugar.activity import activity
 from sugar.graphics import style
