diff --git a/webactivity.py b/webactivity.py
--- a/webactivity.py
+++ b/webactivity.py
@@ -31,9 +31,12 @@ import sqlite3
 import cjson
 import gconf
 
-# HACK: Needed by http://dev.sugarlabs.org/ticket/456
-import gnome
-gnome.init('Hulahop', '1.0')
+try:
+    # HACK: Needed by http://dev.sugarlabs.org/ticket/456
+    import gnome
+    gnome.init('Hulahop', '1.0')
+except:
+    pass
 
 from sugar.activity import activity
 from sugar.graphics import style
